<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Matriz</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(240, 240, 240);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_PrimeiraMatriz">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>1° Matriz</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_QuantLinhas_Matriz_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Quantidade de Linhas </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_QuantColunas_Matriz_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Quantidade de Colunas </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_QuantLinhas_Matriz_1">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>50</y>
      <width>79</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_QuantColunas_Matriz_1">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>90</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Criar_Matriz_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Criar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Limpar_Matriz_1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Exibir_Matriz_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>200</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>5</width>
      <height>300</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_QuantLinhas_Matriz_2">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>50</y>
      <width>78</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_QuantColunas_Matriz_2">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>90</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Exibir_Matriz_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>200</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_QuantLinhas_Matriz_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Quantidade de Linhas </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_QuantColunas_Matriz_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Quantidade de Colunas </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Criar_Matriz_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Criar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Limpar_Matriz_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_PrimeiraMatriz_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>2° Matriz</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>420</width>
      <height>5</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Potenciacao_Matriz_1">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>281</y>
      <width>46</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>../incone_Potencia.jpg</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_MultiplicarPorConstante_Matriz_1">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>281</y>
      <width>45</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>../incone_X_Constante.jpg</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>17</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Expoente_Matriz_1">
    <property name="geometry">
     <rect>
      <x>63</x>
      <y>282</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Constante_Matriz_1">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>282</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>201</width>
      <height>5</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>201</width>
      <height>5</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Constante_Matriz_2">
    <property name="geometry">
     <rect>
      <x>382</x>
      <y>280</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Potenciacao_Matriz_2">
    <property name="geometry">
     <rect>
      <x>228</x>
      <y>279</y>
      <width>46</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../incone_Potencia.jpg</normaloff>../incone_Potencia.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_MultiplicarPorConstante_Matriz_2">
    <property name="geometry">
     <rect>
      <x>331</x>
      <y>279</y>
      <width>45</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>../incone_X_Constante.jpg</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>17</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Expoente_Matriz_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>321</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">gridline-color: rgb(240, 240, 240);
background-color: rgb(240, 240, 240);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="autoScrollMargin">
     <number>20</number>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>true</bool>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>8</number>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <row>
     <property name="text">
      <string>Triangular Superior</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
     <property name="background">
      <color>
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Triangular Inferior</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Simetrica</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Identidade</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Iguais</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Diferentes</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Ortogonal</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Permutação</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>1° Matriz</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="background">
      <color>
       <red>240</red>
       <green>240</green>
       <blue>240</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>2° Matriz</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="background">
      <color>
       <red>240</red>
       <green>240</green>
       <blue>240</blue>
      </color>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>323</y>
      <width>411</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_Adicao">
       <property name="font">
        <font>
         <family>AR CENA</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Subtracao">
       <property name="font">
        <font>
         <family>AR CENA</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Multiplicacao">
       <property name="font">
        <font>
         <family>AR CENA</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Transposta_Matriz_1">
       <property name="font">
        <font>
         <family>AR CENA</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>../incone_Transposta_Matriz_1.jpg</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Transposta_Matriz_2">
       <property name="font">
        <font>
         <family>AR CENA</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(239, 239, 239);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>../incone_Transposta_Matriz_2.jpg</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>label_PrimeiraMatriz</zorder>
   <zorder>label_QuantLinhas_Matriz_1</zorder>
   <zorder>label_QuantColunas_Matriz_1</zorder>
   <zorder>lineEdit_QuantLinhas_Matriz_1</zorder>
   <zorder>lineEdit_QuantColunas_Matriz_1</zorder>
   <zorder>pushButton_Criar_Matriz_1</zorder>
   <zorder>pushButton_Limpar_Matriz_1</zorder>
   <zorder>label_Exibir_Matriz_1</zorder>
   <zorder>line</zorder>
   <zorder>lineEdit_QuantColunas_Matriz_2</zorder>
   <zorder>label_Exibir_Matriz_2</zorder>
   <zorder>label_QuantLinhas_Matriz_2</zorder>
   <zorder>label_QuantColunas_Matriz_2</zorder>
   <zorder>pushButton_Criar_Matriz_2</zorder>
   <zorder>pushButton_Limpar_Matriz_2</zorder>
   <zorder>label_PrimeiraMatriz_2</zorder>
   <zorder>lineEdit_QuantLinhas_Matriz_2</zorder>
   <zorder>line_2</zorder>
   <zorder>pushButton_Potenciacao_Matriz_1</zorder>
   <zorder>pushButton_MultiplicarPorConstante_Matriz_1</zorder>
   <zorder>lineEdit_Expoente_Matriz_1</zorder>
   <zorder>lineEdit_Constante_Matriz_1</zorder>
   <zorder>line_3</zorder>
   <zorder>line_4</zorder>
   <zorder>lineEdit_Constante_Matriz_2</zorder>
   <zorder>pushButton_Potenciacao_Matriz_2</zorder>
   <zorder>pushButton_MultiplicarPorConstante_Matriz_2</zorder>
   <zorder>lineEdit_Expoente_Matriz_2</zorder>
   <zorder>tableWidget</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
